corehq/warehouse/transforms/__init__.py

```diff
@@ -0,0 +1,17 @@
+from corehq.warehouse.const import (
+    APP_STATUS_FACT_SLUG,
+)
+
+
+def app_status_transform(row):
+    print row
```

Comment:
Can we not do the transform prior to putting the data in the staging table since we'll already need to be iterating over each app doc to extract the info?